tchannel/response.py

```diff
@@ -74,17 +75,31 @@ class TransportHeaders(object):
         'scheme',
     )
 
-    def __init__(self,
-                 failure_domain=None,
-                 scheme=None,
-                 **kwargs):
```

Comment:
Yeah but it's safe to assume that people aren't messing with this directly because:

1. It's not a documented public API
2. We don't provide a way to inject one that you construct yourself into a request or response.

We can still add it to the changelog just in case, and maybe release it in a minor version bump.